ethcore/src/spec/spec.rs

```diff
@@ -135,6 +135,12 @@ impl From<ethjson::spec::Spec> for Spec {
 	}
 }
 
+macro_rules! load_bundled {
+	($e:expr) => {
+		Spec::load(include_bytes!(concat!("../../res/", $e, ".json")) as &[u8]).expect("Chain spec is invalid")
```

Comment:
This is purely for us if we decide to add an invalid spec, it will not even compile then.